docs/developer-docs/web-apps/custom-domains/dns-setup.mdx

```diff
@@ -12,109 +12,116 @@ import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+  - #### Step 4: In the next step, Cloudflare lists two nameservers that you should configure GoDaddy to use.
+  Take note of the two nameservers (e.g., `brianna.ns.cloudflare.com` and `kaiser.ns.cloudflare.com`) provided by Cloudflare.
 
-  - #### Step 5: In the **DNS Management** panel of GoDaddy, remove all DNS entries on GoDaddy.
-  Go back to your DNS settings on Godaddy and make sure to remove all the DNS entries that GoDaddy created.
+  - #### Step 5: In the **DNS Management** panel of GoDaddy, remove all DNS entries on GoDaddy.
+  Go back to your DNS settings on Godaddy and make sure to remove all the DNS entries that GoDaddy created.
 
-  - #### Step 6: Click on the **Change** button in the **Nameservers** section.
+  - #### Step 6: Click on the **Change** button in the **Nameservers** section.
 
-  - #### Step 7: In the dialog that opened, click on **Enter my own nameservers (advanced)** and fill in the nameservers from Cloudflare in the provided fields. To finish, click on **Save**.
+  - #### Step 7: In the dialog that opened, click on **Enter my own nameservers (advanced)** and fill in the nameservers from Cloudflare in the provided fields. To finish, click on **Save**.
 
-  ![GoDaddy's dialog to configure the nameservers](godaddy-ns-dialog.png)
+  ![GoDaddy's dialog to configure the nameservers](godaddy-ns-dialog.png)
 
-  - #### Step 8: Confirm in the dialog that you indeed intend to change the nameservers.
-  The resulting configuration should look similar to the following screenshot:
+  - #### Step 8: Confirm in the dialog that you indeed intend to change the nameservers.
+  The resulting configuration should look similar to the following screenshot:
 
-  ![Alternative nameserver configuration on GoDaddy](godaddy-ns-configured.png)
+  ![Alternative nameserver configuration on GoDaddy](godaddy-ns-configured.png)
 
-  - #### Step 9: Back in the Cloudflare management portal, click on **Done, check nameservers**.
-  This step can take several hours and you will be notified by email once it
-  succeeded.
+  - #### Step 9: Back in the Cloudflare management portal, click on **Done, check nameservers**.
+  This step can take several hours, and you will be notified by email once it
+  succeeded.
 
-  - #### Step 10: On Cloudflare, add the following records to configure your domain:
-    * Create a `CNAME` record for which you set the name field to `@` and the
-    target field to `ic-domain.online.icp1.io`;
-    * Create a `CNAME` record for which you set the host field to `_acme-challenge` and the target field to `_acme-challenge.ic-domain.online.icp2.io`;
-    * Create a `TXT` record for which you set the name field to `_canister-id` and the content field to the canister ID `y5jqt-wqaaa-aaaam-abcoq-cai`.
+  - #### Step 10: On Cloudflare, add the following records to configure your domain:
+    * Create a `CNAME` record for which you set the name field to `@` and the
+    target field to `ic-domain.live.icp1.io`;
```

Comment:
```suggestion
    target field to `ic-domain.online.icp1.io`;
```
Same here. I think it is confusing if the screenshot shows a different domain name than the instructions.